test/e2e/page-objects/pages/homepage.ts

```diff
@@ -24,9 +25,12 @@ class HomePage {
 
   public headerNavbar: HeaderNavbar;
 
+  public accountOptionsMenu: AccountOptionsMenu;
+
   constructor(driver: Driver) {
     this.driver = driver;
     this.headerNavbar = new HeaderNavbar(driver);
+    this.accountOptionsMenu = new AccountOptionsMenu(driver);
```

Comment:
since we will not create `accountOptionsMenu` page class, this line can be deleted, `accountOptionsMenu` is a part of `headerNavbar`